app/javascript/article-form/elements/publishToggle.jsx

```diff
@@ -40,7 +40,7 @@ const PublishToggle = ({
       </button>
     )}
     <span>
-      <p style={!edited && { visibility: 'hidden' }}>
+      <p style={!edited && { visibility: 'hidden', cursor: 'pointer' }}>
```

Comment:
I'm not following why we need to change the cursor style on a hidden paragraph.

Reply:
Oh, I thought it changed the cursor on the paragraph when it becomes visible

Reply:
I definitely made a mistake here, I thought the cursor didn't change but trying it on master again everything looks fine.